bootstrap.py

```diff
@@ -127,7 +108,7 @@ def applicable_playbooks():
     """ Returns a list of playbooks that should be applied to this system """
     playbooks = ['']                  # Base Playbook
     playbooks.append(project_path())  # Project Playbook
-    playbooks.extend(role_paths())    # System Roles
+    playbooks.append(role_paths())    # System Roles
```

Comment:
The reason for `extend` vs `append` was clarity. `extend()` requires an iterable, where `append()` just bolts it onto the end, which can behave surprisingly different with iterable and non-iterable items. The way it was written implies that the `project_path()` is expected to return something different than `role_paths()` (which is true, one is expected to be iterable, the other is not).